src/eme.js

```diff
@@ -160,6 +192,19 @@ export const makeNewRequest = (player, requestOptions) => {
           return;
         }
 
+        if (event.messageType === 'license-renewal') {
+          const limitRenewalsBeforePlay = options.limitRenewalsBeforePlay;
+          const limitRenewalsOnPauseTime = options.limitRenewalsOnPauseTime;
+
+          if (!player.hasStarted() && limitRenewalsBeforePlay ||
+            (player.paused() && typeof limitRenewalsOnPauseTime === 'number' && timeElapsed >= limitRenewalsOnPauseTime) ||
+            player.ended()) {
```

Comment:
_nit_: I would split it into several consts eg:
ignoreLinceRenewal (before playback, max pause duration reached, ended)

Reply:
Done!